operator_toolkit/check_mode.py

```diff
@@ -0,0 +1,71 @@
+import argparse
+from http import HTTPStatus
+
+import requests
+from requests.auth import HTTPBasicAuth
+
+from common import (
+    CLI_MESSAGE,
+    sites,
+    AXIOM_API_AUTH_USER,
+    AXIOM_API_AUTH_PASSWORD
+)
+
+
+def check_mode(site, operation_type):
+    """Recieves a site and an operation service and check the current mode on that site.
+
+    :param str site: site id to check
+    :param str operation_type: kind of operation endpoint to check, example: virtual-operator.
+    :return: Returns a report of the site with the current status.
+    :rtype: str.
+    """
+    url = f'http://{operation_type}-{site}.axiomcloud.io/report'
+    try:
+        req = requests.get(
+            url, auth=HTTPBasicAuth(AXIOM_API_AUTH_USER, AXIOM_API_AUTH_PASSWORD)
+        )
+    except Exception as e:
+        return {site: f"INVALID REQUEST! Message: {e}"}
+
+    if req.status_code == HTTPStatus.NOT_FOUND:
+        return {site: f"INVALID REQUEST! Site not found."}
+
+    if req.status_code != HTTPStatus.OK:
+        return {site: f"INVALID REQUEST! Message: {req.json()}"}
+
+    resp = req.json()
+    application_mode = resp['application_mode'] if "application_mode" in resp else None
```

Comment:
```suggestion
    application_mode = resp.get('application_mode', None)
```
Since `resp` is a dictionary and you only want to get the value of the key `application_mode` and if it doesn't exist return None as a value, this approach can be cleaner.